src/Uno.UWP/UI/Composition/Compositor.Skia.cs

```diff
@@ -61,14 +92,19 @@ private static void RenderVisual(SKSurface surface, SKImageInfo info, Visual vis
 
 				if(visual.Clip is InsetClip insetClip)
 				{
-					surface.Canvas.ClipRect(new SKRect {
-						Top = insetClip.TopInset,
-						Bottom = insetClip.BottomInset,
-						Left = insetClip.LeftInset,
-						Right = insetClip.RightInset
-					});
+					var clipRect = new SKRect
+					{
+						Top = insetClip.TopInset - 1,
+						Bottom = insetClip.BottomInset + 1,
+						Left = insetClip.LeftInset - 1,
+						Right = insetClip.RightInset + 1
+					};
+
+					surface.Canvas.ClipRect(clipRect, SKClipOperation.Intersect, true);
 				}
 
+				using var opacityDisposable = PushOpacity(visual.Opacity);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'opacityDisposable'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)